jmix-appsettings/appsettings/src/main/java/io/jmix/appsettings/AppSettingsProperties.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2024 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.appsettings;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.context.properties.bind.DefaultValue;
+
+@ConfigurationProperties(prefix = "jmix.appsettings")
+public class AppSettingsProperties {
+    /**
+     * {@code false} value allows to use settings entities without permissions. <p>
```

Comment:
Don't use a complex formatting here - Idea won't show it properly when you will edit the application.properties. Please follow the format like in the example I mentioned: `io.jmix.datatoolsflowui.DatatoolsUiProperties#entityInfoScriptsEnabled`

For boolean start it with Whether, e.g.:

> Whether the controls to display SQL-scripts for creating / updating / retrieving an entity instance in the Entity Information view is visible